app/modules/individuals/models.py

```diff
@@ -163,6 +163,60 @@ def _ensure_asset_individual_association(self, asset_guid):
                     rt_val = True
         return rt_val
 
+    # 0th individual will be the resultant/remaining individual
+    @classmethod
+    def merge(cls, *individuals, sex=None, primary_name=None):
+        if not individuals or not isinstance(individuals, tuple) or len(individuals) < 2:
+            raise ValueError('must be passed a tuple of at least 2 individuals')
+        target_individual = individuals[0]
+        source_individuals = individuals[1:]
+        data = {
+            'targetIndividualId': str(target_individual.guid),
+            'sourceIndividualIds': [],
+            'sex': sex,
+            'primaryName': primary_name,
+        }
+        for indiv in source_individuals:
+            data['sourceIndividualIds'].append(str(indiv.guid))
+        # response = current_app.edm.request_passthrough_result(
+        response = current_app.edm.request_passthrough(
+            'individual.merge',
+            'post',
+            {
+                'data': data,
+                'headers': {'Content-Type': 'application/json'},
+            },
+            None,
+        )
+        if not response.ok:
+            return response
+
+        result = response.json()['result']
+        assert 'targetId' in result and result['targetId'] == str(target_individual.guid)
+        assert 'merged' in result
+        assert isinstance(result['merged'], dict)
+        assert len(result['merged'].keys()) == len(source_individuals)
+        # first we sanity-check the reported removed individuals vs what was requested
+        for merged_id in result['merged'].keys():
+            assert merged_id in data['sourceIndividualIds']
+            log.info(
+                f"edm reports successful merge of indiv {merged_id} into {result['targetId']} for encounters {result['merged'][merged_id]}; adjusting locally"
+            )
+        # now we steal their encounters and delete them
+        # NOTE:  technically we could iterate over the enc ids in merged.merged_id array, but we run (tiny) risk of this user
+        #   getting assigned to additional encounters in the interim, so instead we just steal all the encounters directly
+        for indiv in source_individuals:
+            for enc in indiv.encounters:
+                log.debug(f'from {indiv}, assigning {enc} to {target_individual}')
```

Comment:
audit log?

Reply:
yeah, i like the idea of tracking this in audit.

Reply:
do it as as a pair of audit_log_object calls passing the two individuals as the obj param as then we can have audit for the deleted and updated individuals making them easier to track